xml/obs_ag_spider_identification.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="_obs_spider_identification">
+ <title>Spider Identification</title>
+ <para>
+   OBS identifies spiders by using an extension called <command>voight_kampff</command>. Spiders will be identified to hide certain
```

Comment:
Nobody cares what this is called or how it is implemented, that is what I meant with my previous comment. Please describe the functionality, the OBS functionality. Something like:

OBS is hiding specific parts/pages of the application from search crawlers (duckduckgo, google etc.), mostly for performance reasons. Which user-agent strings are identified as crawlers is configured in the file
`<filename>/srv/www/obs/api/config/crawler-user-agents.json</filename>`

etc...